Makefile

```diff
@@ -147,7 +147,7 @@ HOSTCFLAGS		+= $(WARNINGS) $(DEFINES) -iquote include/
 export AFLAGS CFLAGS USERCLFAGS HOSTCFLAGS
 
 # Default target
-all: flog criu lib crit
+all: flog criu lib crit amdgpu_plugin
```

Comment:
As Adrian mentioned in his [comment](https://github.com/checkpoint-restore/criu/pull/1519#issuecomment-864825556), the plugin should not be built automatically by typing `make`. This is because it requires additional dependencies that are optional for CRIU.

To enable the build we could use something like `AMDGPU_PLUGIN=1 make` instead?

Reply:
ack, I've added nmk based dependencies in the Makefile